src/sftp.tf

```
@@ -0,0 +1,62 @@
+# To fully configure the storage account, there are some manual steps. See
+# https://pagopa.atlassian.net/wiki/spaces/DEVOPS/pages/511738518/Enable+SFTP+with+Azure+Storage+Account
+
+resource "azurerm_resource_group" "sftp" {
+  name     = "${local.project}-sftp-rg"
+  location = var.location
+}
+
+module "sftp" {
+  source = "git::https://github.com/pagopa/azurerm.git//storage_account?ref=v2.18.0"
+
+  name                = replace("${local.project}-sftp", "-", "")
+  resource_group_name = azurerm_resource_group.sftp.name
+  location            = azurerm_resource_group.sftp.location
+
+  account_kind             = "StorageV2"
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
```

```suggestion
  account_replication_type = var.sftp_account_replication_type
```

sftp_account_replication_type -> LRS for DEV-UAT, GRS for PROD